polymetis/polymetis/tests/python/polymetis/test_torchscript_pinocchio.py

```diff
@@ -46,15 +47,19 @@ def pybullet_env():
 
 @pytest.fixture
 def pinocchio_wrapper():
-    return toco.models.RobotModelPinocchio(urdf_path, "iiwa_joint_ee")
+    return toco.models.RobotModelPinocchio(urdf_path, "panda_link8")
```

Comment:
We're now using the link name of the EE to get the body frame, so maybe we should update the robot_model configs to also use the ee link name instead of the ee joint name? I'm actually doubtful if it'll actually work with the ee joint name.

Reply:
Thanks, fixed